### PR TITLE
Change JavaDocs for 2 class as a sample to standardize

### LINKED ISSUE #152: Add some sample standardized JavaDocs as guide for team

### PR DESCRIPTION
**Pull request by Kair0s3**
Fixes #152

### CODE REVIEW

File: src/main/java/decodex/parser/Parser.java [hunk 1]
```diff
@@ -95,11 +97,11 @@ public String getInputString(String userInput) throws ParserException {
     }
 
     /**
-     * Parses the user input provided by user and returns its respective command. Returns a null object if the command
-     * is invalid.
+     * Parses the user input specified by the user and returns its respective Command object.
      *
      * @param userInput The user input specified by the user.
      * @return The respective Command object.
```
Comment (rizemon) [inline src/main/java/decodex/parser/Parser.java:103]: Likewise, perhaps use the word `corresponding` instead?

**Review by arraysius — CHANGES_REQUESTED** [review]
Great base line docs. Some missing docs and some messages can be simplified.

**Review by rizemon — APPROVED** [review]
LGTM

File: src/main/java/decodex/parser/Parser.java [hunk 2]
```diff
@@ -128,52 +130,50 @@ public Command parseCommand(String userInput) throws ParserException {
...
     }
 
     /**
-     * Crafts and returns the Data Command given the user input.
+     * Crafts and returns the InputCommand object using the specified user input.
      *
      * @param userInput The user input specified by the user.
-     * @return The Data command
-     * @throws ParserException ParserException
+     * @return The InputCommand object.
```
[the remark this answers is not in the document]
Comment (arraysius) [reply src/main/java/decodex/parser/Parser.java:145]: `@throws` for `ParserException` is missing.

**Pull request merged**